sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_requests.py

```diff
@@ -7,7 +7,9 @@
 import hashlib
 import base64
 import hmac
+from yarl import URL
 from azure.core.pipeline.policies import HTTPPolicy
+from azure.core.pipeline.transport import AioHttpTransport
```

Comment:
Why sync-requests relies on async transport?

Reply:
Does this import incur an `aiohttp` dependency when using synchronous code? If so, I wonder if we should split up the auth policy into a sync and async file...

Reply:
I don't see async requests file, like `_azure_appconfiguration_requests_async.py`, so I think sync and async are combined together, we should split them.

Reply:
In most cases, we don't need to differentiate between sync requests and async requests.

Reply:
Do you mean it's not worth to split from this point?

Reply:
I don't think we need to split it. But we need to make sure sync client works correctly if aiohttp is not installed.

Reply:
Yes, sync client works well because all sync tests are passing in live and playback without the updates here.

Reply:
Can you double check if aiohttp library is installed when running sync tests?

It looks to me they will not pass if it is not installed.

Reply:
You are right, sync tests won't work without aiohttp.

Reply:
Ah, I forgot that `azure-storage-blob` also accounted for this in their auth policy by doing a conditional import: https://github.com/Azure/azure-sdk-for-python/blob/d1ab444d9be412b9e93aaeb22b7df1f8a765e360/sdk/storage/azure-storage-blob/azure/storage/blob/_shared/authentication.py#L22-L25

We should probably do the same -- I think splitting into sync and async policies would be overkill if we can avoid it.